src/main/java/seedu/address/logic/commands/listcommand/ListParticipantCommand.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.logic.commands.listcommand;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashMap;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+
+public class ListParticipantCommand extends ListCommand {
+
+    /* Possible Fields? */
+    public static final String MESSAGE_SUCCESS = "Listed all participants";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+
+        model.getParticipantList().stream()
```

Comment:
Just checking, do you want `getParticipantList()` to return a stream or a list? Could implement returning a stream on my end if you want it.

Reply:
I think `getParticipantList()` should just return a `ParticipantList`. It's basically a getter for Model class so I don't think we should return a list or stream with this method.

Reply:
Oh yea btw just noticed this also. It's `model.getParticipantList().list().stream()`